corehq/apps/userreports/expressions/getters.py

```diff
@@ -75,6 +76,10 @@ def transform_date(item):
     return item or None
 
 
+def transform_datetime(item):
+    return item or None
```

Comment:
yeah something in the DB layer handles this, but agree. don't think it's urgent to fix but would probably be better to do it eventually